doc/examples/applications/plot_tool_using_holoviz.py

```diff
@@ -0,0 +1,202 @@
+"""
+==========================================================
+Explore and annotate image data interactively with HoloViz
+==========================================================
+
+In this tutorial, we build interactive tools and an application using
+`Panel <https://panel.holoviz.org>`_ and `HoloViews <https://holoviews.org>`_.
+
+We will build an application based on the Sobel algorithm to interactively
+find a great color map for the visualization. We will add some nice features,
+like a download button, to make sharing your findings easier.
+
+These techniques can make your analyses more interactive and, hence, powerful.

```

Comment:
```suggestion
Interactive applications provide flexible ways to visualize and explore data,
making data analysis more powerful and, at the same time, accessible to
non-technical audiences.
```
You might as well stick with a very generic sentence.